guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -4,74 +4,19 @@
 
 Use this procedure to enable the repositories that are required to install {ProductName}. Choose from the available list which operating system and version you are installing on:
 
-ifdef::foreman-el,katello[]
-* xref:#repositories-centos-8[CentOS 8]
-* xref:#repositories-centos-7[CentOS 7]
-endif::[]
 ifdef::foreman-el,katello,satellite[]
 * xref:#repositories-rhel-8[{RHEL} 8]
 * xref:#repositories-rhel-7[{RHEL} 7]
 endif::[]
+ifdef::foreman-el,katello[]
+* xref:#repositories-centos-8[CentOS 8]
+* xref:#repositories-centos-7[CentOS 7]
+endif::[]
```

Comment:
Why are you moving those four lines?

Reply:
Initially, I had in my mind that RHEL should come first in sequencing. But, dropped that.